package.json

```diff
@@ -12,7 +12,7 @@
   "author": "elastic.io GmbH",
   "license": "Apache License 2.0",
   "dependencies": {
-    "elasticio-sailor-nodejs": "2.5.1",
+    "elasticio-sailor-nodejs": "2.6.0",
```

Comment:
requires node 12. Right now this component uses is 10.6

Reply:
Changed